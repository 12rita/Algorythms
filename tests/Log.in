log(2, x)
log(x, 2)
log(x, x)
log(arcsin(x), (x^2+3*x+4))
ln(x)
ln(2)