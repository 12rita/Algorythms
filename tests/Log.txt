log(2,x)
log(x,2)
log(x,x)
ln(x)
ln(2)