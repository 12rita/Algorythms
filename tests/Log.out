1/(x*ln(2)
-1/(log(2,x))^2*(1/(x*ln(2))
0
((1/((x^2+3*x+4))*(2*x^(2-1)+3+0))*ln(arcsin(x))-(1/(arcsin(x))*(1/(1-(x)^2)^(1/2)))*ln((x^2+3*x+4)))/ln(arcsin(x))^2
1/(x)
0
